src/app/components/AddButton/AddButton.tsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import classes from './AddButton.module.css';
 import { GiLightBackpack } from 'react-icons/gi';
+import { Link } from 'react-router-dom';
 
 export default function AddButton(): JSX.Element {
   return (
     <div
       data-augmented-ui="all-hexangle-up border"
       className={classes.button_area_div}
     >
-      <button onClick={alert} className={classes.add_button_symbol}>
-        <GiLightBackpack size={42} />
-      </button>
+      <Link to="/input">
+        <button className={classes.add_button_symbol}>
+          <GiLightBackpack size={42} />
+        </button>
+      </Link>
```

Comment:
Instead of wrapping an HTML button with react router link you can use the useNavigate hook. Maybe that makes more sense for you.
https://reach.tech/router/api/useNavigate